src/components/project/project.service.ts

```diff
@@ -74,105 +75,201 @@ export class ProjectService {
-        WITH * OPTIONAL MATCH (requestingUser)<-[:member]-(canEditEstimatedSubmission:ACL {canEditEstimatedSubmission: true})-[:toNode]->(project)
-
-        WITH * OPTIONAL MATCH (requestingUser)<-[:member]-(canReadModifiedAt:ACL {canReadModifiedAt: true})-[:toNode]->(project)-[:modifiedAt {active: true}]->(modifiedAt:Property {active: true})
-        WITH * OPTIONAL MATCH (requestingUser)<-[:member]-(canEditModifiedAt:ACL {canEditModifiedAt: true})-[:toNode]->(project)
-
-        RETURN
-          project.id as id,
-          project.createdAt as createdAt,
-          type.value as type,
-          sensitivity.value as sensitivity,
-          name.value as name,
-          deptId.value as deptId,
-          step.value as step,
-          status.value as status,
-          country,
-          region,
-          zone,
-          mouStart.value as mouStart,
-          mouEnd.value as mouEnd,
-          estimatedSubmission.value as estimatedSubmission,
-          modifiedAt.value as modifiedAt,
-          canReadType.canReadType as canReadType,
-          canEditType.canEditType as canEditType,
-          canReadSensitivity.canReadSensitivity as canReadSensitivity,
-          canEditSensitivity.canEditSensitivity as canEditSensitivity,
-          canReadName.canReadName as canReadName,
-          canEditName.canEditName as canEditName,
-          canReadDeptId.canReadDeptId as canReadDeptId,
-          canEditDeptId.canEditDeptId as canEditDeptId,
-          canReadStep.canReadStep as canReadStep,
-          canEditStep.canEditStep as canEditStep,
-          canReadStatus.canReadStatus as canReadStatus,
-          canEditStatus.canEditStatus as canEditStatus,
-          canReadLocation.canReadLocation as canReadLocation,
-          canEditLocation.canEditLocation as canEditLocation,
-          canReadMouStart.canReadMouStart as canReadMouStart,
-          canEditMouStart.canEditMouStart as canEditMouStart,
-          canReadMouEnd.canReadMouEnd as canReadMouEnd,
-          canEditMouEnd.canEditMouEnd as canEditMouEnd,
-          canReadEstimatedSubmission.canReadEstimatedSubmission as canReadEstimatedSubmission,
-          canEditEstimatedSubmission.canEditEstimatedSubmission as canEditEstimatedSubmission,
-          canReadModifiedAt.canReadModifiedAt as canReadModifiedAt,
-          canEditModifiedAt.canEditModifiedAt as canEditModifiedAt
-      `,
-        {
-          token: session.token,
-          requestingUserId: session.userId,
-          owningOrgId: session.owningOrgId,
-          id,
-        }
-      )
-      .first();
+        relation('out', '', 'baseNode', { active: true }),
+        node('project'),
+        relation('out', '', 'location', { active: true }),
+        node('location', 'location', { active: true }),
```

Comment:
does we have `location` label ? I guess its `Country`

Reply:
Yes, I changed to Country